infra/gravity/config.go

```diff
@@ -92,6 +94,8 @@ type ProvisionerConfig struct {
 	storageDriver StorageDriver
 	// dockerDevice is a physical volume where docker data would be stored
 	dockerDevice string `validate:"required"`
+	// waitDisks is whether to wait for disks to "spin up"
+	waitDisks bool
```

Comment:
Aslo, `LoadConfig` on line 115 uses invalid `Fatal` call with values and a format string (`Fatalf` is expected)